src/editors/hooks.ts

```diff
@@ -6,10 +6,11 @@ import analyticsEvt from './data/constants/analyticsEvt';
 import { actions, thunkActions } from './data/redux';
 import { RequestKeys } from './data/constants/requests';
 
+
 // eslint-disable-next-line react-hooks/rules-of-hooks
 export const initializeApp = ({ dispatch, data }) => useEffect(
   () => dispatch(thunkActions.app.initialize(data)),
-  [data],
+  [data?.blockId],
```

Comment:
To be safe, I think you could do `[data?.studioEndpointUrl, data?.blockId, data?.learningContextId]`

Reply:
Fixed!